data-serving/scripts/convert_data.py

```diff
@@ -0,0 +1,51 @@
+# Lint as: python3
+'''Script to convert CSV line-list data into json compliant with the MongoDB schema.'''
+
+import argparse
+import csv
+import logging
+import json
+import pandas as pd
+from pandas import DataFrame
+from typing import List
+
+
+def main():
+    logging.basicConfig(filename='convert_data.log',
+                        filemode='w', level=logging.DEBUG)
+
+    parser = argparse.ArgumentParser(
+        description='Convert CSV line-list data into json compliant with the MongoDB schema.')
+    parser.add_argument('--csv_path', required=True)
+    parser.add_argument('--json_path', default='cases.json')
+    args = parser.parse_args()
+    logging.info("Args: %s", args)
+
+    print('Reading data from', args.csv_path)
+    original_cases = read_csv(args.csv_path)
+
+    print('Converting data to new schema')
+    converted_cases = convert(original_cases)
+
+    print('Writing results to', args.json_path)
+    write_json(converted_cases, args.json_path)
+
+    print('Great success! 🎉')
+
+
+def read_csv(csv_path: str) -> DataFrame:
+    return pd.read_csv(csv_path, header=0, low_memory=False)
```

Comment:
encoding='utf-8'

always, you'll thank me later

Reply:
done, thanks!

Reply:
no?